### ISSUE TITLE
2D plots with latitude and height

### ISSUE THREAD

**Issue by doutriaux1** [UV-CDAT/uvcmetrics#57]
originally posted by @potter2 here:
https://github.com/UV-CDAT/uvcdat/issues/1001

**Comment by painter1** [comment]
@potter2, should level *always* be in hPa?

**Comment by potter2** [comment]
I think it should. I would rather not have the actual log but just a log scale. At least it should default to hPa. Having height in km on the right is a nice touch too.

Thanks,
Jerry


Gerald (Jerry) Potter
NASA Goddard Space Flight Center
E-mail  gerald.potter@nasa.gov
Office  28:S241
Phone    707.315.8314
Analysis and Data Consultant
Department      GSFC:606.2

**Comment by susburrows** [comment]
I agree that it should be in hPa, but I vote for labelling the axis "Pressure" rather than "Level" (since in general it is not really the model level, and the units are pressure units).

**Comment by potter2** [comment]
I agree it should be in hPa.
